tutorials/getting_started/configuration.md

```diff
@@ -0,0 +1,344 @@
+that indicates how to embed
+the tokens in that particular namespace. The `TextFieldEmbedder`'s output is the concatenation
+of these embeddings.
+
+The `"tokens"` namespace (which consists of integer encodings of the lowercased words in the input)
+gets fed into an
+[`Embedding`](http://docs.allennlp.org/en/latest/api/allennlp.modules.token_embedders.html?highlight=embedding#allennlp.modules.token_embedders.embedding.Embedding)
+module that embeds the vocabulary words in a 50-dimensional space.
+
+The `"token_characters"` namespace (which consists of integer-sequence encodings of the characters in each word)
+gets fed into a
+[`TokenCharactersEncoder`](http://docs.allennlp.org/en/latest/api/allennlp.modules.token_embedders.html?highlight=embedding#allennlp.modules.token_embedders.token_characters_encoder.TokenCharactersEncoder),
+which embeds the characters in an 8-dimensional space
+and then applies a
+[`CnnEncoder`](http://docs.allennlp.org/en/latest/api/allennlp.modules.seq2vec_encoders.html#allennlp.modules.seq2vec_encoders.cnn_encoder.CnnEncoder)
+that uses 50 filters and so also produces a 50-dimensional output. You can see that this encoder also uses a 20% dropout during training.
+
+The output of this `TextFieldEmbedder` is a 50-dimensional vector for `"tokens"`
+concatenated with a 50-dimensional vector for `"token_characters`";
+that is, a 100-dimensional vector.
+
+### The Seq2SeqEncoder
+
+The output of that embedding is processed by our `Seq2SeqEncoder`:
+
+```js
+    "stacked_encoder": {
+            "type": "lstm",
+            "input_size": 100,
+            "hidden_size": 100,
+            "num_layers": 2,
+            "dropout": 0.5,
+            "bidirectional": true
+    }
+```
+
+The `"lstm"` encoder is just a thin wrapper around
+[`torch.nn.LSTM`](http://pytorch.org/docs/master/nn.html#torch.nn.LSTM),
+and its parameters are simply passed through to the PyTorch constructor.
+Its input size needs to match the 100-dimensional output size of the
+previous embedding layer.
+
+And, as mentioned above, the output of this layer gets passed to a linear layer
+that doesn't need any configuration. That's all for the model.
+
+## Training the Model
+
+The rest of the config file is dedicated to the training process.
+
+```js
+  "iterator": {"type": "basic", "batch_size": 32},
+```
+
+We'll iterate over our datasets using a
+[`BasicIterator`](http://docs.allennlp.org/en/latest/api/allennlp.data.iterators.html#allennlp.data.iterators.basic_iterator.BasicIterator)
+that pads our data and processes it in batches of size 32.
+
+```js
+  "optimizer": "adam",
+```
+
+We'll optimize using
+[`torch.optim.Adam`](http://pytorch.org/docs/master/optim.html#torch.optim.Adam)
+with its default parameters.
+
+```js
+  "trainer": {
+    "num_epochs": 40,
+    "patience": 10,
+    "cuda_device": -1
+  }
+}
+```
+
+Finally, we'll run the training for 40 epochs;
+we'll stop prematurely if we get no improvement for 10 epochs;
+and we'll train on the CPU.
```

Comment:
You might add instruction (or point somewhere) for how to specify a GPU, as this field is a bit opaque.